src/physics/em/PhotoelectricInteractor.hh

```diff
@@ -0,0 +1,102 @@
+//----------------------------------*-C++-*----------------------------------//
+// Copyright 2020 UT-Battelle, LLC, and other Celeritas developers.
+// See the top-level COPYRIGHT file for details.
+// SPDX-License-Identifier: (Apache-2.0 OR MIT)
+//---------------------------------------------------------------------------//
+//! \file PhotoelectricInteractor.hh
+//---------------------------------------------------------------------------//
+#pragma once
+
+#include "base/Algorithms.hh"
+#include "base/Macros.hh"
+#include "base/Types.hh"
+#include "physics/base/Interaction.hh"
+#include "physics/base/ParticleTrackView.hh"
+#include "physics/base/SecondaryAllocatorView.hh"
+#include "physics/base/Secondary.hh"
+#include "physics/base/Units.hh"
+#include "PhotoelectricInteractorPointers.hh"
+#include "PhotoelectricMicroXsCalculator.hh"
+
+namespace celeritas
+{
+//---------------------------------------------------------------------------//
+/*!
+ * Livermore model for the photoelectric effect.
+ *
+ * A parameterization of the photoelectric cross sections in two different
+ * energy intervals, formulated as \sigma(E) = a_1 / E + a_2 / E^2 + a_3 / E^3
+ * + a_4 / E^4 + a_5 / E^5 + a_6 / E^6, is used. The coefficients for this
+ * model are calculated by fitting the tabulated EPICS2014 subshell cross
+ * sections. The parameterized model applies above approximately 5 keV; below
+ * this limit (which depends on the atomic number) the tabulated cross sections
+ * are used. The angle of the emitted photoelectron is sampled from the
+ * Sauter-Gavrila distribution.
+ *
+ * \note This performs the same sampling routine as in Geant4's
+ * G4LivermorePhotoElectricModel class, as documented in section 6.3.5 of the
+ * Geant4 Physics Reference (release 10.6).
+ */
+class PhotoelectricInteractor
```

Comment:
Oh no you're right, I figured we'd need to adjust the name. I was hesitant to call it `LivermoreInteractor` since there's a "Livermore model" for each of the photon processes, but there are a few different models for the photoelectric effect. Maybe something like `LivermorePE` or `LivermorePhotoelectric`...